xbmc/music/Album.cpp

```diff
@@ -176,6 +176,28 @@ std::string CAlbum::GetGenreString() const
   return StringUtils::Join(genre, g_advancedSettings.m_musicItemSeparator);
 }
 
+const std::vector<std::string>& CAlbum::GetAlbumArtist() const
+{
+  //Get artist names as vector from artist credits
+  std::vector<std::string> albumartists;
+  for (VECARTISTCREDITS::const_iterator artistCredit = artistCredits.begin(); artistCredit != artistCredits.end(); ++artistCredit)
+  {
+    albumartists.push_back(artistCredit->GetArtist());
+  }
+  return albumartists;
```

Comment:
Thanks, I like to learn new things. But not sure if back_inserter will work here as I am copying the string part of an array of objects into an array of strings, rather than just copying one vector to another. To be honest I am not keen on fancy methods either, rather have more lines of code that is easy to read and understand (as long as it is not too slow). But happy to follow the Kodi standard. Anyone?